ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -195,12 +209,18 @@ bracketChainSyncClient ::
     -- It's more important that the client is removed from the set promptly
     -- than it is for the client to be added promptly, because of how this is
     -- used by the GSM to determine that the node is done syncing.
+ -> StrictTVar m (Map peer (ChainSyncClientHandle m blk))
  -> peer
  -> NodeToNodeVersion
  -> ChainSyncLoPBucketConfig
  -> (     StrictTVar m (AnchoredFragment (Header blk))
+          -- ^ Variable holding the current fragment
        -> (m (), m ())
        -> (m (), m (), m ())
```

Comment:
These parameters are related to LoP. It would be nice to explain what they are here.